nb/api/Wave.md

```diff
@@ -0,0 +1,20 @@
+### Wave()
+Parameter|Default|Type
+---|---|---
+op|Point|Function or shape to build from.
```

Comment:
Is this new?

Reply:
No, it's been around for a long time, but it's only sometimes useful in generating profiles.